### DISCUSSION TITLE
First time contributions message is wrong

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#49594] [Mobile] [answered]
GitHub Mobile beta
Browse to
https://github.com/desktop/desktop/pull/16277
Expand the checks section

![Screenshot_20230309-013114.png](https://user-images.githubusercontent.com/2119212/223939766-cfc67ee0-d339-4a26-a687-fee6d55a9104.png)

> Checks awaiting approval
> First-time contributors require maintainers to approve workflows in order to run.

This is grammatically wrong. I am a first-time contributor, and it isn't my policy that's requiring approval, it's the maintainers' policy that requires approval for requests from people like me. 

The simplest fix is something like this:
```diff
-First-time contributors require maintainers to approve workflows in order to run.
+First-time contributor contributions require maintainer approval in order for workflows to run.
```

This is still a passive voice which is grammatically awkward.

Grammatically better would be something like:
> Maintainers require approval for workflows in pull requests from first time contributors.

That's an active voice, as the maintainers are actively declared as being responsible for the policy that is impacting an object (workflows in pull requests) that in turn happen to have been created by first-time contributors.

It's true that this wording doesn't indicate who is responsible for providing that approval, but, first-time contributors will be able to intuit that they themselves can't provide it, and a logical conclusion would be that it's the repository maintains who can grant approval.

Technically the policy might be enforced at an organization or higher level, but repository maintainers choose to maintain their repository in the organization, so they're implicitly requiring any rules dictated by their organization. If they don't like their organization's policy, they could maintain a fork with different rules (and cease to maintain the one with rules they don't like).

**Comment by aconsuegra** [comment, marked as answer]
Hi @jsoref, thanks for your feedback! I'll create an issue for our team to fix this copy, I agree that it sounds wrong.
